config.json

```diff
@@ -1,5 +1,6 @@
 {
-  "secret": "secret",
+  "token_hmac_secret_key": "secret",
+  "token_rsa_public_key": "-----BEGIN PUBLIC KEY-----\nMFwwDQYJKoZIhvcNAQEBBQADSwAwSAJBAJ+A48YVz4XVzEIHEQAYdmBg9Y4yaNhu\nMTAfeZ9BZRaYTNNMloE0MGLII4aZukRjiW0pjT96i1PRezH7jk10HzcCAwEAAQ==\n-----END PUBLIC KEY-----",
```

Comment:
I suppose we can remove this from config example because it does not have many sense without knowing private key